python/jsbeautifier/javascript/beautifier.py

```diff
@@ -583,6 +599,11 @@ def handle_end_expr(self, current_token):
                 current_token, current_token.text == ']' and self.is_array(
                     self._flags.mode) and not self._options.keep_array_indentation)
 
+        if self._options.brace_style == "expand-all" and \
+                current_token.text == ']' and \
+                self.is_array(self._flags.mode):
+            self.print_newline()
+
```

Comment:
New feature to put ending square bracket on its own separate line (Newline before)